src/PKSim.Core/Snapshots/Mappers/ParameterMapper.cs

```diff
@@ -0,0 +1,37 @@
+﻿using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Formulas;
+using SnapshotParameter = PKSim.Core.Snapshots.Parameter;
+using SnapshotTableFormula = PKSim.Core.Snapshots.TableFormula;
+using ModelTableFormula = OSPSuite.Core.Domain.Formulas.TableFormula;
+
+namespace PKSim.Core.Snapshots.Mappers
+{
+   public class ParameterMapper : SnapshotMapperBase<IParameter, SnapshotParameter>
+   {
+      private readonly TableFormulaMapper _tableFormulaMapper;
+
+      public ParameterMapper(TableFormulaMapper tableFormulaMapper)
+      {
+         _tableFormulaMapper = tableFormulaMapper;
+      }
+
+      public override SnapshotParameter MapToSnapshot(IParameter modelParameter)
+      {
+         var snapshotParameter = new SnapshotParameter();
+         MapBaseProperties(modelParameter, snapshotParameter);
+         //override description that might have been set since parameter in PK-Sim all come from DB
+         snapshotParameter.Description = null;
```

Comment:
@Yuri05 We do not want to serialize the PK-Sim parameter description (or do we??) since they all come from the database. But we can decide otherwise easily

Reply:
Agree. Description should not be saved